plugins/woocommerce-admin/client/inbox-panel/test/index.js

```diff
@@ -114,10 +149,67 @@ describe( 'inbox_panel_view_event', () => {
 			notesHaveResolved: true,
 			isBatchUpdating: false,
 		} ) );
+		InboxNoteCard.mockImplementation( ( { note } ) => {
+			return <div>{ note.id }</div>;
+		} );
 		render( <InboxPanel /> );
 
 		expect( recordEvent ).toHaveBeenCalledWith( 'inbox_panel_view', {
 			total: 5,
 		} );
 	} );
 } );
+
+describe( 'inbox_note_view event', () => {
+	test( 'should fire when inbox note card calls onNoteVisible', () => {
+		notes.forEach( ( note ) => ( note.is_deleted = false ) );
+		useSelect.mockImplementation( () => ( {
+			notes,
+			isError: false,
+			notesHaveResolved: true,
+			isBatchUpdating: false,
+		} ) );
+		InboxNoteCard.mockImplementation( ( { onNoteVisible, note } ) => {
+			useEffect( () => onNoteVisible( note ), [] );
+			return <div>{ note.id }</div>;
+		} );
+		render( <InboxPanel /> );
+		notes.forEach( ( note ) => {
+			expect( recordEvent ).toHaveBeenCalledWith( 'inbox_note_view', {
+				note_content: note.content,
+				note_name: note.name,
+				note_title: note.title,
+				note_type: note.type,
+				screen: '',
+			} );
+		} );
```

Comment:
Why do we check that each note logs a Tracks event?

Are we really testing anything specific to multiple notes here if we are mocking how notes become visible (assuming that it what is being done)?